starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <head>
+    <title>Apple Pie Recipe</title>
+    <link rel="stylesheet" href="./css/main.css">
+  </head>
+  <body>
+    <h1>Apple Pies</h1>
```

Comment:
here you could wrap your h1 element with a <header> tag, just for semantic purposes ;)